### PR TITLE
WWST-5862 Added a FP for the ABL Light

### PR DESCRIPTION
**Pull request by chw-choi**
@lukhong

### CODE REVIEW

**Comment by lukhong** [comment]
@greens @tpmanley Please review this new DTH. I added some information on the ticket.

**Comment by greens** [comment]
Why is this a separate DTH?

**Comment by lukhong** [comment]
@greens LED team has checked with adding an FP and changed commit here.

File: devicetypes/smartthings/zigbee-white-color-temperature-bulb.src/zigbee-white-color-temperature-bulb.groovy [hunk 1]
```diff
@@ -92,6 +92,9 @@ metadata {
 		// Third Reality
 		fingerprint profileId: "0104", inClusters: "0000, 0003, 0004, 0005, 0006, 0008, 0300", outClusters: "0019", manufacturer: "Third Reality, Inc", model: "3RSL011Z", deviceJoinName: "RealityLight"
 		fingerprint profileId: "0104", inClusters: "0000, 0003, 0004, 0005, 0006, 0008, 0300", outClusters: "0019", manufacturer: "Third Reality, Inc", model: "3RSL012Z", deviceJoinName: "RealityLight"
+
+		// ABL
```
Comment (tpmanley) [inline devicetypes/smartthings/zigbee-white-color-temperature-bulb.src/zigbee-white-color-temperature-bulb.groovy]: Please move this fingerprint group up so it's in alphabetical order. Also remove the spaces before the colons.

File: devicetypes/smartthings/zigbee-white-color-temperature-bulb.src/zigbee-white-color-temperature-bulb.groovy [hunk 2]
```diff
@@ -34,6 +34,9 @@ metadata {
 		// Generic
 		fingerprint profileId: "0104", deviceId: "010C", inClusters: "0006, 0008, 0300", deviceJoinName: "Generic Color Temperature Light"
 
+		// ABL
+		fingerprint profileId: "0104", inClusters: "0000, 0003, 0004, 0005, 0006, 0008, 0300", outClusters: "0019", manufacturer: "Samsung Electronics", model: "ABL-LIGHT-Z-001", deviceJoinName: "Ultra Thin Wafer", mnmn: "Samsung Electronics", vid: "ABL-LIGHT-Z-001", ocfDeviceType: "oic.d.light"
```
Comment (tpmanley) [inline devicetypes/smartthings/zigbee-white-color-temperature-bulb.src/zigbee-white-color-temperature-bulb.groovy]: It isn't necessary to set `ocfDeviceType` at the fingerprint level. All the devices that use this DTH are lights and the `"oic.d.light"` type is applied to every device using this DTH due to the existence of the `capability "Light"`.

**Review by tpmanley — APPROVED** [review]

**Pull request merged**